### ISSUE TITLE
This is awesome + trackpads on both sides?

### ISSUE THREAD

**Issue by teije** [Rookwork/saucer-diy-voyager-trackpad#1]
Hi Robin! 

Like you, I was working on a project, one that involved a 40mm Cirque trackpad and somehow mounting it to my Voyager. 
You can't imagine how mind blown I was when I read the ZSA newsletter! To see someone had already done it, and took a better approach than I was going to (I wanted to use an Arduino to act as a HID device).

So, first of all: thanks for your hard work, the Saucer looks amazing.
I have ordered the splitter and 

One question I have is: 

Do you know if it is possible to use two trackpads? 

I was hoping I could mount one trackpad to each half.
Maybe I can wire up two trackpads to one sets of wires of the spltiter? I am not at all familiar with QMK and I2C, so I was hoping I could ask you for some insight.

**Comment by Rookwork** [comment]
Hello! I'm glad you like the project. :) A funny aside about your question: I tried this as I was experimenting with housing designs. I had two trackpads and decided to connect them both just to see what would happen, expecting nothing. To my surprise, it actually worked... kind of. Amusingly, both trackpads kind of fought for control and made the cursor very unpredictable. Not exactly the most productive way to use them, but it was nevertheless neat that they both kind of worked right away without any special configuration.  

I didn't end up actually doing this, but I'm pretty confident all you would need to do to fix this is assign a unique address to each trackpad. I am admittedly not familiar enough with the deeper implementation of sensors in QMK to tell you exactly how to do this, but if you ask on the QMK Discord, I'm sure they'll be able to help. Drashna in particular would be one to ask. 

Hope that helps!

**Comment by teije** [comment]
Awesome! I am glad to hear this is probably viable. 

Did the trackpads work properly and in a predictible manner when only one is used at a time?

Also, I was wondering what your process is when making changes to your layout. Do you still use the ZSA browser editor, then "Download Source" and then apply the firmware changes as described in your repo's readme (https://github.com/Rookwork/saucer-diy-voyager-trackpad)?

**Comment by Rookwork** [comment]
Yup! You can freely switch one side at a time if you want to do that — I didn't see any issues. 

And right, I edit my layout in Oryx and manually apply the trackpad code then compile locally. There is this neat new tool that I just came across, though: https://github.com/poulainpi/oryx-with-custom-qmk. I experimented with adding trackpad code specifically and it worked great. This makes the whole thing much more seamless.
